atef/ui/any_value_widget.ui

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>Form</class>
+ <widget class="QWidget" name="Form">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>400</width>
+    <height>300</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Form</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <property name="leftMargin">
+    <number>0</number>
+   </property>
+   <property name="topMargin">
+    <number>0</number>
+   </property>
+   <property name="rightMargin">
+    <number>0</number>
+   </property>
+   <property name="bottomMargin">
+    <number>0</number>
+   </property>
+   <item>
+    <widget class="QTableWidget" name="values_table">
```

Comment:
Would recommend this setting for these table widgets for a low-effort way to fill the space a bit better:
<img width="738" alt="image" src="https://user-images.githubusercontent.com/5139267/193872901-8d4b8953-a7b3-4c78-8e80-3a9925209d49.png">

Reply:
This ends up being a pretty major simplification for a lot of my extra event handlers/row spacing and gives a better result at the end